src/test/java/com/library/steps/US06_stepDefinitions.java

```diff
@@ -0,0 +1,95 @@
+package com.library.steps;
+import com.library.pages.BookPage;
+import com.library.pages.DashBoardPage;
+import com.library.pages.LoginPage;
+import com.library.utility.BrowserUtil;
+import com.library.utility.ConfigurationReader;
+import com.library.utility.DB_Util;
+import com.library.utility.Driver;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+import java.util.Map;
+
+public class US06_stepDefinitions {
+
+
+  LoginPage loginPage=new LoginPage();
+  DashBoardPage dashBoardPage=new DashBoardPage();
+
+  BookPage bookPage=new BookPage();
+    @Given("the {string} on the home page")
+    public void the_on_the_home_page(String user) {
+      Driver.getDriver().get(ConfigurationReader.getProperty("library_url"));
```

Comment:
Hi Noor, you don't need line 25, line 26 will bring you to home page

for line 26 
loginPage.login(string);    -->> you can pass string because it'c parameterize it auto link with feature that we have  "librarian"  so again only 

loginPage.login(string);     ---> will work , please try and let's me know

Reply:
Thank you login method has been fixed!